data/mods/Medieval_Stuff/armor.json

```diff
@@ -0,0 +1,294 @@
+        "to_hit" : 0,
+        "storage" : 0,
+        "symbol" : "[",
+        "description" : "An early medieval helmet with a projection guarding the nose, to better protect the face without obstructing vision.  Proper viking gear, unlike the stereotypical horned helm.",
+        "price" : 26000,
+        "material" : ["iron", "null"],
+        "volume" : 8,
+        "cutting" : 0,
+        "warmth" : 10,
+        "phase" : "solid",
+        "environmental_protection" : 0,
+        "encumbrance" : 10,
+        "bashing" : 10,
+        "flags" : ["VARSIZE", "STURDY"],
+        "coverage" : 80,
+        "material_thickness" : 3
+    },{
+        "type" : "ARMOR",
+        "id" : "helmet_conical",
+        "category" : "armor",
+        "name" : "conical helm",
+        "weight" : 1425,
+        "color" : "brown",
+        "covers" : ["HEAD"],
+        "techniques" : ["WBLOCK_1"],
+        "to_hit" : 0,
+        "storage" : 0,
+        "symbol" : "[",
+        "description" : "A conical iron helm with additional protection for the neck, associated with the Mongol Empire.",
+        "price" : 40000,
+        "material" : ["iron", "fur"],
+        "volume" : 14,
+        "cutting" : 0,
+        "warmth" : 35,
+        "phase" : "solid",
+        "environmental_protection" : 1,
+        "encumbrance" : 20,
+        "bashing" : 6,
+        "flags" : ["STURDY"],
+        "coverage" : 90,
+        "material_thickness" : 3
+    },{
+        "type" : "ARMOR",
+        "id" : "tunic",
+        "name" : "tunic",
+        "name_plural" : "tunics",
+        "weight" : 70,
+        "color" : "dark_gray",
+        "covers" : ["LEGS", "TORSO", "ARMS"],
+        "to_hit" : -5,
+        "storage" : 0,
+        "symbol" : "[",
+        "description" : "A simple garment long enough to cover down to the knees.",
+        "price" : 4200,
+        "material" : ["cotton", "null"],
+        "volume" : 5,
+        "cutting" : 0,
+        "warmth" : 2,
+        "phase" : "solid",
+        "environmental_protection" : 0,
+        "encumbrance" : 0,
+        "bashing" : -5,
+        "flags" : ["VARSIZE"],
+        "coverage" : 75,
+        "material_thickness" : 1
+    },{
+        "type" : "ARMOR",
+        "id" : "gambeson",
+        "name" : "gambeson",
+        "weight" : 453,
+        "color" : "light_gray",
+        "covers" : ["TORSO", "ARMS", "LEGS"],
+        "to_hit" : 0,
+        "storage" : 0,
+        "symbol" : "[",
+        "description" : "A thick jacket of quited fabric, designed to be worn underneath mail or other armor.  Or worn on its own, if you can't afford proper armor.",
+        "price" : 20000,
+        "material" : ["cotton", "null"],
+        "volume" : 8,
+        "cutting" : 0,
+        "warmth" : 30,
+        "phase" : "solid",
+        "environmental_protection" : 0,
+        "encumbrance" : 10,
+        "bashing" : -2,
+        "flags" : ["VARSIZE", "SKINTIGHT", "STURDY"],
+        "coverage" : 80,
+        "material_thickness" : 5
```

Comment:
That's 8/8 protection at 5 encumbrance (when fit), skintight. That 80% coverage isn't enough to make it anything but too good. OK in a mod, but unacceptably good in mainline.